<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionBase</class>
 <widget class="QWidget" name="QuestionBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="prevQuestion">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>590</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>←  上一頁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextQuestion">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>590</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下一頁  →</string>
   </property>
  </widget>
  <widget class="QLabel" name="correctState">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="progress">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
